contrib/gcppubsub/config/300-gcppubsubsource.yaml

```diff
@@ -48,7 +48,7 @@ spec:
               description: "Credential to use to poll the GCP PubSub Subscription. It is not used to create or delete the Subscription, only to poll it. The value of the secret entry must be a service account key in the JSON format (see https://cloud.google.com/iam/docs/creating-managing-service-account-keys)."
             googleCloudProject:
               type: string
-              description: "ID of the Google Cloud Project that the PubSub Topic exists in."
+              description: "ID of the Google Cloud Project that the PubSub Topic exists in. E.g. 'my-project-1234' rather than its display name, 'My Project' or its number '1234567890'."
             serviceAccountName:
               type: string
               description: "Name of the ServiceAccount that will be used to run the Receive Adapter Deployment."
```

Comment:
re: Adam's comment "Give an example. Is this the full email address? Just the name before the '@'?"

I'm not sure, it was just a copy/paste from the types.go comment. Is it a full email address?

Reply:
My mistake on the earlier comment. This field is about the K8s Service Account, not any GCP service account. How about:

```
Name of the Kubernetes ServiceAccount that is running the Receiver Adapter Pods.
```

Reply:
Done!